docs/contract-rate-non-revision-design.md

```diff
@@ -0,0 +1,77 @@
+# Contract and Rate Without Revisions Design
+
+This is an ER diagram of the data that is planned to be sent to Salesforce on submission. It relies on Salesforce's native diff tracking so elides the concept of Revisions. 
+
+```mermaid
+erDiagram
+
+Contract {
+    String id
+    DateTime initiallySubmittedAt
+    DateTime updatedAt
+    StateCode stateCode
+    String name
+    String mccrsID
+
+    String[] programNicknames
```

Comment:
I'm assuming we will need more robust programs data - the full names and the nicknames. We have different UI we display in our app already depending on context, sometimes we refer to programs by acronym sometimes its the full name written out.

Reply:
We talked about this in a meeting, I'm not actually sure we surface the long names anywhere in our app right now. I was thinking that just the program short names would be fine for now